core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -516,7 +516,7 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 			}
 
 			// Add the name of the parent elements
-			if ($strTable && \in_array($strTable, $arrTables) && $strTable != $arrTables[0])
+			if ($strTable && \in_array($strTable, $arrTables) && $strTable != ($arrTables[0] ?? null))
```

Comment:
How can `\in_array($strTable, $arrTables)` be `true` if `$arrTables[0]` is not set?

Reply:
You're right, wrongly parsed the parentheses in my head.

Reply:
Reverted in 36e3bda